ovos_persona_server/ollama.py

```
@@ -371,3 +372,150 @@ async def tags(persona: Persona = Depends(get_default_persona)) -> OllamaTagsRes
+        request_body: Push request with model name (accepted, ignored).
+
+    Returns:
+        Success status stub.
+    """
+    return JSONResponse({"status": "success"})
+
+
+@ollama_router.post("/embeddings")
+async def ollama_embeddings(
+        request_body: OllamaEmbedRequest,
+        persona: Persona = Depends(get_default_persona),
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)